server/src/main/java/org/elasticsearch/action/search/OpenPointInTimeResponse.java

```diff
@@ -12,40 +12,55 @@
 import org.elasticsearch.common.ParseField;
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ConstructingObjectParser;
+import org.elasticsearch.common.xcontent.ObjectParser;
 import org.elasticsearch.common.xcontent.ToXContentObject;
 import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentParser;
 
 import java.io.IOException;
 import java.util.Objects;
 
+import static org.elasticsearch.common.xcontent.ConstructingObjectParser.constructorArg;
```

Comment:
It is good to add `OpenPointInTimeResponseTests` to check serialization from and to XContent. But I don't mind if we do it in a following PR.

Reply:
Yes, please see https://github.com/elastic/elasticsearch/pull/72167#discussion_r631389330